src/Mvp/Crud/CrudView.php

```diff
@@ -30,7 +30,19 @@
 
 class CrudView extends HtmlView
 {
-    public $newModel = true;
+    protected $title;
+    protected $newModel = true;
+
+    public function __construct()
+    {
+        $this->attachEventHandler("SetTitle", function ($title) {
```

Comment:
Events going from presenter to view - I wonder if that's too odd? Normally our events all run one way from view to presenter (and then to parent view etc.)

If we could achieve the same this by calling $this->raiseEvent( "GetTitle" ) in the view when it's needed, and attach the event handler in the ConfigureView function of the presenter, it would be consistent then with all other raiseEvents in MVP. Thoughts?